iamproperty/iamproperty.github.io
# Ds 42/lists

## Summary of Changes:
1. List documentation

## Rationale
Document the lists in the framework

## How to Test
Open up the preview site (see the below netlify comment below) and navigate to /elements/lists

## Ticket(s)
DS-42

## Checklist

## Notes
...
